src/components/Input/Input.scss

```
@@ -0,0 +1,18 @@
+input {
+  display: flex;
+  height: 40px;
+  width: 100%;
+  border-radius: 0px;
+  border-color: #DEDFE1;
+  border-style: solid;
+
+  &.input__text{
+    &--center{

```

since you don't have any other class inside the `input__text` toy can just use direct:

```suggestion
  &.input__text--center {
```